apps/aws-amplify/actions/amplify.ts

```diff
@@ -0,0 +1,21 @@
+import { AppActionCallContext } from '@contentful/node-apps-toolkit';
+
+interface AppActionCallParameters {
+  amplifyWebhookUrl: string;
+}
+
+export const handler = async (payload: AppActionCallParameters, context: AppActionCallContext) => {
+  const { amplifyWebhookUrl } = payload;
+  try {
+    await fetch(amplifyWebhookUrl, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
```

Comment:
post w/out body intentional/how it works yeah?

Reply:
I would have expected that `AppActionCallParameters` or *something* was passed through

Reply:
Yep it is intentional, could pass an empty `{}` but wasn't necessary, absolutely 0 info passed/recieved from triggering the build though